### DISCUSSION TITLE
Feature Request: Dark Mode

### DISCUSSION THREAD

**Discussion by pedrorj2** [pdfslick/pdfslick#43] [Ideas]
It would be great to have an integrated dark mode option. Thanks for this repo, works great!

**Comment by van100j** [comment]
Hi @pedrorj2! 

PDFSlick leaves the app implementation details to developers, so you should be able to add dark mode on your own. 

I've created a [quick Svelte app](https://github.com/pdfslick/pdfslick/blob/main/apps/svelteweb/src/routes/dark-mode/%2Bpage.svelte) ([demo](https://svelte.pdfslick.dev/dark-mode)) which allows switching between light/dark theme based on the system preferences.

https://github.com/pdfslick/pdfslick/assets/408278/298d5781-1ebc-48ec-918a-c8dbcc4173fd

Hope this helps.

**Reply by pedrorj2** [reply]
Thank you so much for the response! I am just starting with web development from scratch now that I’ve finished my college exams, and a PDF viewer like this is what I needed for a little project that I have, so I really appreciate it!

This is something I’ll be working on over the next few months, so I’ll be figuring out how to implement it in my app :)

Thanks again!

**Reply by van100j** [reply]
No problem! Best of luck on your project!
